README.md

```diff
@@ -204,78 +225,69 @@ env = gym.make(
     visdom=True,
 )
 ```
-
-### Interfacing w/ PyMARL and malib
-
-[PyMARL](https://github.com/oxwhirl/pymarl) and [malib](https://github.com/ying-wen/malib) have been open-sourced. You can run them via,
-
+Below is a sample visualization of an agent's camera sensor observations.
+<p align="center">
+<img src="docs/_static/visdom.gif" ><br/>
```

Comment:
Would you happen to have any idea why the observations are so noisy? It looks like there is some issue with the height, colour estimation, or FoV of the camera that is causing the image to distort.

Reply:
Nope.